examples/next/pages/index.js

```diff
@@ -0,0 +1,52 @@
+import 'carbon-components/scss/globals/scss/styles.scss';
+
+import React, { Component } from 'react';
+import { Accordion, AccordionItem } from 'carbon-components-react';
+
+import '../styles/App.scss';
+
+class App extends Component {
+  render() {
+    return (
+      <>
+        <div className="App">
+          <header className="App-header">
+            <h1 className="App-title">
+              Welcome to React, with Carbon and Next!
+            </h1>
+          </header>
+          <p className="App-intro">
+            To get started, edit <code>pages/index.js</code> and save to reload.
+            Then, check out the{' '}
+            <a href="https://www.nexjs.org/learn">Next.js</a> tutorial.
```

Comment:
Uh - I guess I meant here instead of what I pointed out earlier. You meant `nextjs.org` with (`t`)?